config/initializers/huginn_agent.rb

```diff
@@ -0,0 +1,5 @@
+require 'huginn_agent'
+
+HuginnAgent.hack_huginn_to_accept_me
```

Comment:
This "hack" is just a couple monkey-patches to the Agent model.  They just make sure that the agents built using this gem are incorporated in any lists of agents.

https://github.com/darrencauthon/huginn_agent/blob/a66a5de153c05fc7cb667e0f8014c45dfdd05a7a/lib/huginn_agent/hacker.rb#L7